services/ai-navigator-app/0.1.0/ai-navigator-app.yaml

```diff
@@ -0,0 +1,52 @@
+# TODO: Should these be separate files?
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ai-navigator-app-config
+  namespace: kommander
+data:
+  CHATBOT_URL: "https://chatbot.api.d2iq.com/v1/chat"
+  AUTHORIZER_URL: "https://auth.api.d2iq.com"
+  LICENSE_NAMESPACE: "kommander" # TODO: use kommander namspace, is this hardcoded ok?
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ai-navigator-app
+  namespace: kommander # TODO: use kommander namspace, is this hardcoded ok?
+  labels:
+    app: ai-navigator-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ai-navigator-app
+  template:
+    metadata:
+      labels:
+        app: ai-navigator-app
+    spec:
+      containers:
```

Comment:
> What do I need to do so that the service would be able to read licenses and the corresponding secret?

you will need a `Role`, `RoleBinding`, and a `ServiceAccount`. then you apply that service account to this deployment. 

This is a simple example: https://github.com/mesosphere/kommander-applications/blob/main/services/grafana-loki/0.69.16/grafana-loki-pre-install-jobs/pre-install.yaml

Reply:
Done and tested ✅